pages/task/read/content/review.js

```diff
@@ -0,0 +1,14 @@
+const { merge } = require('lodash');
+const baseContent = require('./base');
+const refusalNotice = require('./refusal-notice');
+
+module.exports = merge({}, baseContent, {
+  refusalNotice,
+  page: {
+    heading: 'Check notice of intention to refuse',
+    summary: 'This notice will be sent to the applicant and a copy forwarded to the establishment licence holder and admins.'
```

Comment:
I don't know if I missed something in the notifications PR, but how does this notice actually get sent to those people?

Reply:
There's a notification to external users when the task status is updated to intention-to-refuse:
https://github.com/UKHomeOffice/asl-notifications/blob/master/lib/recipients/project.js#L294-L305

and another notification to the inspector when the refusal deadline expires:
https://github.com/UKHomeOffice/asl-notifications/blob/master/lib/recipients/project.js#L144-L160

which is triggered by:
https://github.com/UKHomeOffice/asl-workflow/blob/master/bin/project-refusal-deadline-passed